src/figuresClasses.ts

```diff
@@ -1,11 +1,81 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('The sides of the triangle must be greater than zero.');
+    }
+
+    const longest: number = Math.max(this.a, this.b, this.c);
+    const sumOfOther:number = this.a + this.b + this.c - longest;
+
+        if (longest >= sumOfOther) {
+          throw new Error(
+            'The longest side should be less than the sum of the other.',
+          );
+        }
+  }
+
+  getArea(): number {
+
+
+    const p = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(p * (p - this.a) * (p - this.b) * (p - this.c));
 
-export class Triangle implements Figure {}
+    return Math.round(area * 100) / 100;
```

Comment:
```suggestion
    return Math.floor(area * 100) / 100;
```

In this case as you see, you use same like of code in two places, move it to some function and reuse